controllers/registry/package/changes.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var packageService = require('../../../services/package');
+
+// GET /-/_changes?since={timestamp}&limit={number}&cursorId={number}
+// List packages names since the timestamp
+// Similar with https://replicate.npmjs.com/_changes?since=7139538
+module.exports = function* listSince() {
+  var query = this.query;
+  var since = query.since || '0';
+  var limit = Number(query.limit);
```

Comment:
if `limit === 0`, no tag will be listed.

Reply:
If the `limit` is specified directly,
It should be the expected.